Проект 4ого спринта Гарейшина Дарья

GareyshinaDV opened the pull request.

GareyshinaDV committed:
Проект 4ого спринта

pomegranate282922 commented on src/test/java/orderTest.java:
```diff
@@ -0,0 +1,98 @@
+import pageObject.OrderPage;
+
+@RunWith(Parameterized.class)
+public class orderTest {
```
Классы именуем начиная с заглавной буквы

pomegranate282922 commented on src/test/java/orderTest.java:
```diff
@@ -0,0 +1,98 @@
+        ChromeOptions options = new ChromeOptions();
+        driver = new ChromeDriver(options);
+        // Переход на страницу тестового приложения заказа Самоката
+        driver.get("https://qa-scooter.praktikum-services.ru/");
```
Урлы лучше выносить как константы. Так легче их переиспользовать и изменять.

pomegranate282922 commented on src/test/java/orderTest.java:
```diff
@@ -0,0 +1,98 @@
+    }
+
+    @Test  // Тест прохождения позитивного сцерания при заказе через кнопку "Заказать" с хедере главной страницы
+    public void OrderTestTopButton() {
```
Методы именуем начиная со строчной буквы. Нужно исправить везде.

pomegranate282922 commented on src/test/java/pageObject/MainPage.java:
```diff
@@ -0,0 +1,43 @@
+package pageObject;
```
Пакеты именуем строчными буквами без camelCase

pomegranate282922 reviewed (commented):
Хорошая работа!
Тесты сделаны верны, pom тоже грамотно реализована.
Есть замечания к неймингу - прошу их поправить по всему коду.

GareyshinaDV committed:
Проект 4ого спринта. Исправления по ревью:
1) добавила класс с константой URL;
2) поправила наименования классов, пакетов и методов.

pomegranate282922 commented on pom.xml:
```diff
@@ -0,0 +1,40 @@
+
+    <dependencies>
+        <dependency>
+            <groupId>com.fasterxml.jackson.core</groupId>
```
Зависимость выглядит лишней.

pomegranate282922 reviewed (approved):
Супер, зачёт!
